contracts/simpleKitty/SimpleKittyCore.sol

```diff
@@ -0,0 +1,38 @@
+pragma solidity 0.4.24;
+
+import "./SimpleKittyOwnership.sol";
+
+contract SimpleKittyCore is SimpleKittyOwnership {
+    /// @notice Returns all the relevant information about a specific kitty.
+    /// @param _id The ID of the kitty of interest.
+    function getKitty(uint256 _id)
+        external
+        view
+        returns (
+            bool isGestating,
+            bool isReady,
+            uint256 cooldownIndex,
+            uint256 nextActionAt,
+            uint256 siringWithId,
+            uint256 birthTime,
+            uint256 matronId,
+            uint256 sireId,
+            uint256 generation,
+            uint256 genes
+        )
+    {
+        Kitty storage kit = kitties[_id];
+
+        // if this variable is 0 then it's not gestating
+        isGestating = (kit.siringWithId != 0);
+        isReady = (kit.cooldownEndBlock <= block.number);
```

Comment:
I suggest to return `false` here since this check must be only available for actual Kitty rather than its reflection in a sidechain.